### PR TITLE
[bitnami/nginx] Allow setting deployment's revisionHistoryLimit

### PR DESCRIPTION
**Pull request by kforsthoevel**
### Description of the change

This PR allows setting the deployment's `revisionHistoryLimit` to another value than the implicit default value of 10 for the Bitnami nginx chart.

### Benefits

Users of can set the deployment's `revisionHistoryLimit` to another value than the implicit default value of 10.

### Possible drawbacks

None.

### Additional information

None.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/nginx/values.yaml [hunk 1]
```diff
@@ -264,7 +267,8 @@ resources:
 ## If you do want to specify lifecycleHooks, uncomment the following
 ## lines, adjust them as necessary, and remove the curly braces on 'lifecycle:{}'.
 ## @param lifecycleHooks Optional lifecycleHooks for the NGINX container
-lifecycleHooks: {}
+lifecycleHooks:
+  {}
```
Comment (joancafom) [inline bitnami/nginx/values.yaml:271]: Please don't modify these as well.

File: bitnami/nginx/README.md [hunk 2]
```diff
@@ -410,9 +411,9 @@ On 9 April 2022, security vulnerabilities in the [NGINX LDAP reference implement
 #### What changes were introduced in this major version?
 
 - Previous versions of this Helm Chart use `apiVersion: v1` (installable by both Helm 2 and 3), this Helm Chart was updated to `apiVersion: v2` (installable by Helm 3 only). [Here](https://helm.sh/docs/topics/charts/#the-apiversion-field) you can find more information about the `apiVersion` field.
-- Move dependency information from the *requirements.yaml* to the *Chart.yaml*
-- After running `helm dependency update`, a *Chart.lock* file is generated containing the same structure used in the previous *requirements.lock*
-- The different fields present in the *Chart.yaml* file has been ordered alphabetically in a homogeneous way for all the Bitnami Helm Charts
+- Move dependency information from the _requirements.yaml_ to the _Chart.yaml_
+- After running `helm dependency update`, a _Chart.lock_ file is generated containing the same structure used in the previous _requirements.lock_
+- The different fields present in the _Chart.yaml_ file has been ordered alphabetically in a homogeneous way for all the Bitnami Helm Charts
```
Comment (joancafom) [inline bitnami/nginx/README.md:416]: This as well

File: bitnami/nginx/README.md [hunk 3]
```diff
@@ -397,7 +398,7 @@ Affected values:
 
 This major release no longer uses the bitnami/nginx-ldap-auth-daemon container as a dependency since its upstream project is not actively maintained.
 
-*2022-04-12 edit*:
+_2022-04-12 edit_:
```
Comment (joancafom) [inline bitnami/nginx/README.md:401]: I guess this is caused by some formatter you have installed in your system. Could you please remove these changes from your commit?

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for your changes! I left some minor comments for you to apply before proceeding to merge the PR :)

**Comment by kforsthoevel** [comment]
I have removed the formatting changes.

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for the PR 🚀

**Pull request merged**